packages/sonarwhal/.sonarwhalrc

```diff
@@ -41,4 +41,4 @@
         "validate-set-cookie-header": "warning",
         "x-content-type-options": "warning"
     }
-}
+}
```

Comment:
Can you add back the empty trailing line in this file so no changes apply to the `sonarwhal` package?

Reply:
There was no empty line. I [copied everything from master](https://raw.githubusercontent.com/sonarwhal/sonarwhal/master/packages/sonarwhal/.sonarwhalrc) and pasted in this file. So it should be same as master ?